app/classifier/components/Intervention/Intervention.spec.js

```diff
@@ -30,7 +30,14 @@ describe('Intervention', function () {
   });
   it('should show a notification message', () => {
     const newlineMsg = intervention.message + "\n"
-    expect(wrapper.find(Markdown).text()).to.equal(newlineMsg);
+    expect(wrapper.find(Markdown).first().text()).to.equal(newlineMsg);
+  });
+  it('should show an opt-out message', () => {
+    const expectedMarkdown = 'I do not want to take part in this messaging [study]' +
```

Comment:
You can also get this string by loading the English language file, but that might be a lot of work for a single string.

Reply:
I really want to make sure that the test has the study link setup, if we use the translation strings themselves and someone removes the link in the translation string the tests will pass but without the correct study link.

Reply:
That makes sense, but we're now maintaining the text in two places, which is slightly annoying.